src/Shopify.php

```diff
@@ -580,7 +677,9 @@ private static function makeUri($api, $ids = [], $queue = [], $append = '', $bas
             $endpoint = implode('/', array_filter($parent)).'/'.$endpoint;
         }
 
-        $endpoint = '/'.$base.'/'.$endpoint;
+        $endpoint = Util::isLaravel()
```

Comment:
@shinyhiram Just ate about 90 minutes trying to figure out what happened here.

Luckily, I'm also using this project on a personal project on your latest version.

Requests were failing with weirdness.

TLDR. `config('shopify.api_base')` was intended to take the version (if any). So the new `config('shopify.api_version')`.

In addition to that, my env (personal app) and production, already set the api_version in `config('shopify.api_base')` so I had to punch myself in the face until I figure out it was an old env var breaking shit.

<img width="920" alt="Screen Shot 2020-04-25 at 9 13 02 PM" src="https://user-images.githubusercontent.com/470255/80294825-b9022580-873a-11ea-8169-a621184ba206.png">

<img width="643" alt="Screen Shot 2020-04-25 at 9 16 15 PM" src="https://user-images.githubusercontent.com/470255/80294826-bf909d00-873a-11ea-9858-e7fb8c5154f9.png">